### PR TITLE
VAGOV-1738: Adding excluded type logic for previews.

### PR DESCRIPTION
**Pull request by ethanteague**
## What this does
  - Removes "Preview" button from node view page for 5 node types.

## QA
  - Go to `admin/content`
  - Visit the node view page and confirm "Preview" button does not appear for 1 of each type:

1. outreach_asset
2. person_profile
3. support_service
4. regional_health_care_service_des
5. health_care_local_health_service

### CODE REVIEW

File: docroot/modules/custom/va_gov_build_trigger/va_gov_build_trigger.module [hunk 1]
```diff
@@ -87,9 +87,17 @@ function va_gov_build_trigger_preprocess_page(&$variables) {
...
-    if (empty($arg[1]) || empty($arg[2]) || $arg[1] !== 'node' && !is_numeric($arg[2])) {
+    $exclusion_types = [
+      'outreach_asset',
+      'person_profile',
+      'support_service',
+      'regional_health_care_service_des',
+      'health_care_local_health_service',
+    ];
+    // Make sure we aren't on the node form or an excluded type.
+    if ((empty($arg[1]) || empty($arg[2]) || $arg[1] !== 'node') &&
```
Comment (swirtSJW) [inline docroot/modules/custom/va_gov_build_trigger/va_gov_build_trigger.module]: This is kind of an old school method for determining the route.  I think we want to use a routname here instead

```
$route_name = \Drupal::routeMatch()->getRouteName();
if ($route_name !== 'entity.node.edit_form') {

}
```

Comment (ethanteague) [reply docroot/modules/custom/va_gov_build_trigger/va_gov_build_trigger.module]: That's a lot nicer, @swirtSJW :+1: - updated.

**Comment by ElijahLynn** [comment]
> Step 18 start: Create header and footer
> 
> /var/aegir/projects/cms/pr876/docroot/vendor/va-gov/web/src/site/stages/build/index.js:245
>     if (err) throw err;
>              ^
> 
> TypeError: Cannot read property 'data' of undefined
>     at formatHeaderData (/var/aegir/projects/cms/pr876/docroot/vendor/va-gov/web/src/site/stages/build/drupal/menus.js:267:3

We should `composer update va-gov/web` I think. We are supposed to have this updated automatically with dependabot but it is broken right now due to an upstream issue here https://github.com/dependabot/feedback/issues/803.

**Comment by swirtSJW** [comment]
Passed php unit Locally
image


Passed behat locally 
image

Can no longer wait on devshop, merging based on local tests passing.

**Pull request merged**